frameworks/Java/micronaut/common-test/src/main/resources/fortunes.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>
+    <tr><td>0</td><td>Additional fortune added at request time.</td></tr>
+    <tr><td>1</td><td>message0</td></tr>
+    <tr><td>2</td><td>message1</td></tr>
+    <tr><td>3</td><td>message2</td></tr>
+    <tr><td>4</td><td>message3</td></tr>
+    <tr><td>5</td><td>message4</td></tr>
+    <tr><td>6</td><td>message5</td></tr>
+    <tr><td>7</td><td>message6</td></tr>
+    <tr><td>8</td><td>message7</td></tr>
+    <tr><td>9</td><td>message8</td></tr>
+    <tr><td>10</td><td>message9</td></tr>
+</table></body></html>
```

Comment:
This template looks like it assumes that there will be exactly 10 fortunes. Fortune templates need to be ready for any size list. Future verifications will test this by changing the size of the fortunes.

Reply:
I don't think the template is actually used the one that is used is https://github.com/TechEmpower/FrameworkBenchmarks/blob/c7cbcf9723aa6c48e6b784a5a6631efc1b48b381/frameworks/Java/micronaut/common/src/main/resources/views/fortunes.rocker.html

@dstepanov can you delete this one?

Reply:
No, it’s used for the unit testing validation